.github/workflows/publish-github.yml

```diff
@@ -0,0 +1,54 @@
+name: Publish
```

Comment:
Should this file just be named `publish.yml`?

Reply:
it was originally named `publish.yml` but we now have a secondary publishing workflow so i decided to break it out into two different workflows for github/pmc which i named appropriately